src/shogun/classifier/svm/NewtonSVM.cpp

```diff
@@ -127,7 +127,7 @@ void CNewtonSVM::iteration()
 
 	linalg::add(Xsv2sum, lcrossdiag, Xsv2sum);
 	SGMatrix<float64_t> inverse((x_d + 1), (x_d + 1));
-	SGMatrix<float64_t>::pinv(Xsv2sum.data(), x_d + 1, x_d + 1, inverse.data());
```

Comment:
you could use the other interface here if you re-allocate the inverse in every iteration (no need to do 2 lines for that)
But in fact since the size is constant, why dont you share the memory of that matrix across iterations. Then it would be a bit more efficient...

Reply:
it is only used in newtonsvm it seems :)

Reply:
ok kill it then :)